bindings/matrix-sdk-crypto-nodejs/README.md

```diff
@@ -184,6 +184,20 @@ See
 [`tracing-subscriber`](https://tracing.rs/tracing_subscriber/index.html)
 to learn more about the `RUST_LOG`/`MATRIX_LOG` environment variable.
 
+#### Using tracing in a development environment
+
+To use tracing in client applications that import these bindings, here's how to do it in
+a local development environment:
+
+-   In this directory, run `npm link` to make your local build of the bindings available to
+    other Node projects on your system
+-   In your client app's source directory, run `npm link @matrix-org/matrix-sdk-crypto-nodejs`
+    to make it use your trace-enabled local build of the bindings
+-   In your client app's source code, add a call to `initTracing` near startup time
+-   Run your app with the `MATRIX_LOG` environment variable set to the desired log level
```

Comment:
```suggestion
- In this directory, run `npm link` to make your local build of the bindings available to
  other Node projects on your system
- In your client app's source directory, run `npm link @matrix-org/matrix-sdk-crypto-nodejs`
  to make it use your trace-enabled local build of the bindings
- In your client app's source code, add a call to `initTracing` near startup time
- Run your app with the `MATRIX_LOG` environment variable set to the desired log level
```

Reply:
I don't think the whole `npm link` stuff is important here, is it? Or maybe it is important if `matrix-sdk-crypto-nodejs` is a dependency of another project?

Reply:
It's the latter.  Should the section title be changed to make that more clear?